README.md

```diff
@@ -81,11 +113,27 @@ The Kibana default configuration is stored in `kibana/config/kibana.yml`.
 
 It is also possible to map the entire `config` directory instead of a single file.
 
-## How can I tune the Logstash configuration?
+## How can I tune Logstash configuration?
```

Comment:
This whole section was updated via https://github.com/deviantony/docker-elk/commit/ac945e0c96129aeb273f2ebd79d4dbafc79dc15c

Why would you want to revert the changes?